packages/hdwallet-ledger/src/ethereum.ts

```diff
@@ -150,7 +150,9 @@ export async function ethSignMessage(
   msg: core.ETHSignMessage
 ): Promise<core.ETHSignedMessage> {
   const bip32path = core.addressNListToBIP32(msg.addressNList);
-  const res = await transport.call("Eth", "signPersonalMessage", bip32path, Buffer.from(msg.message).toString("hex"));
+
+  const buffer = typeof msg.message === "string" ? Buffer.from(msg.message) : Buffer.from(new Uint8Array(msg.message));
```

Comment:
If possible, we should use `ethers.utils.arrayify`/ `core.arrayify` instead of `Uint8Array`, which effectively does the same but is a battle-tested, nice abstraction to pack things

Reply:
With that being said, not sure if this really matters in this module, `@shapeshiftoss/hdwallet-ledger` isn't used by our projects, and I'm not sure it's maintained. @pastaghost might have some context on whether or not it is (supposed to be) used by the community

Reply:
@gomesalexandre I don't think we can use `ethers.utils.arrayify/ core.arrayify` here since msg.message is already of types `Bytes`. Those methods are expecting a string as input. 
We are simply transforming the `Bytes` input to an `Uint8Array` which is pretty straightforward I think.